packages/firestore/test/unit/api/database.test.ts

```diff
@@ -0,0 +1,151 @@
+/**
+ * Copyright 2017 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { expect } from 'chai';
+import {
+  CollectionReference,
+  DocumentReference,
+  DocumentSnapshot,
+  Query,
+  QuerySnapshot
+} from '../../../src/api/database';
+import {
+  collectionReference,
+  documentReference,
+  documentSnapshot,
+  query,
+  querySnapshot
+} from '../../util/api_helpers';
+import { expectEqual, expectNotEqual } from '../../util/helpers';
+
+describe('CollectionReference', () => {
+  it('CollectionReference equality checks', () => {
```

Comment:
The way the describe() and it() work is they get concatenated together and are supposed to essentially form a sentence, e.g.:

```
describe('CollectionReferences', () => {
  it('support equality checking with isEqual()', () => {
    ...
  })
})
```

and then when the tests run you'll see

CollectionReferences support equality testing with isEqual() PASSED

We're not super rigorous about it forming a real sentence, but you should at least avoid duplicating 'CollectionReference' in the describe() and the it().